content/07-native-tokens/03-getting-started.mdx

```diff
@@ -154,29 +158,28 @@ This minting policy requires any transaction that mints tokens to be witnessed b
 
 #### Overview
 
-This section describes how to manually mint a new native token ('melcoin') using cardano-cli, and send a transaction of this newly minted token to a new address. The code
-used throughout pertains to the *Mary* Testnet. For the mainnet, replace `--network-magic 3` with `--mainnet` in all commands.
+This section describes how to manually mint a new native token ('melcoin') using cardano-cli, and send a transaction of this newly minted token to a new address. 
  
 #### Pre-requisites 
 
-1. Download the node and config files for the Mary testnet (Launchpad) using this code:
+1. Download the node and config files for the public testnet using this code:
 
 ```bash
-wget https://hydra.iohk.io/build/5266641/download/1/cardano-node-1.24.2-linux.tar.gz
-tar xzvf cardano-node-1.24.2-linux.tar.gz
+wget https://hydra.iohk.io/build/16159630/download/1/cardano-node-1.35.0-linux.tar.gz
+tar xzvf cardano-node-1.35.0-linux.tar.gz
 mkdir lpconfig && cd lpconfig
-wget https://hydra.iohk.io/build/5102327/download/1/launchpad-config.json
-wget https://hydra.iohk.io/build/5102327/download/1/launchpad-byron-genesis.json
-wget https://hydra.iohk.io/build/5102327/download/1/launchpad-shelley-genesis.json
-wget https://hydra.iohk.io/build/5102327/download/1/launchpad-topology.json
+wget https://hydra.iohk.io/build/13695229/download/1/testnet-config.json
+wget https://hydra.iohk.io/build/13695229/download/1/testnet-byron-genesis.json
+wget https://hydra.iohk.io/build/13695229/download/1/testnet-shelley-genesis.json
+wget https://hydra.iohk.io/build/13695229/download/1/testnet-topology.json
```

Comment:
We shouldn't put static links to builds and configs here because they'll soon be out of date.
We already have a Prerequisites section above, so it's best to point to the [releases page](https://github.com/input-output-hk/cardano-node/releases) for latest node build and point to [The Cardano Book](https://book.world.dev.cardano.org/environments.html) for config files there.